DataFormats/Detectors/GlobalTracking/src/RecoContainer.cxx

```diff
@@ -1180,7 +1180,12 @@ gsl::span<const o2::trd::CalibratedTracklet> RecoContainer::getTRDCalibratedTrac
 
 gsl::span<const o2::trd::TriggerRecord> RecoContainer::getTRDTriggerRecords() const
 {
-  return inputsTRD->mTriggerRecords;
+  if (inputsTRD == nullptr) {
+    LOG(ERROR) << "No TRD triggers";
```

Comment:
if the TRD is not there, this is not an error, you can probably log a warning once, no need to do this at every TF.

Reply:
so I added warning counter to RecoContainr. h. OK ?

Reply:
yes, but it seems you did not push it.

Reply:
I pushed. Hmm there is problem - I try to increase counter in const function:
https://github.com/lietava/AliceO2/blob/a68ed6692d911e8da791b4f4751510500af938f2/DataFormats/Detectors/GlobalTracking/src/RecoContainer.cxx#L1186
Shall I give up this message ?
R.

Reply:
you can use local static counter.